.github/workflows/maven.yml

```
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        java: [8, 11, 14, 15, 16, 17-ea]
+        java: [8, 11]
```

I'd leave 16 in and turn that into 17 later in September.
Rationale: Test on all LTS and latest-GA versions. Recommended by Oracle and IBM.
```suggestion
        java: [8, 11, 16]
```